code/game/machinery/iv_drip.dm

```diff
@@ -593,7 +595,18 @@
 	breather = null
 	update_icon()
 
+/obj/machinery/iv_drip/proc/tank_on()
+	playsound(src, 'sound/effects/internals.ogg', 100)
+	tank.forceMove(breather)
+	breather.internal = tank
+	if(breather.internals)
+		breather.internals.icon_state = "internal1"
+	valve_open = TRUE
+	update_icon()
+	return
+
 /obj/machinery/iv_drip/proc/tank_off()
+	playsound(src, 'sound/effects/internals.ogg', 100)
```

Comment:
this does not match the behaviour of back-worn internals, where turning off internals does not play a sound, iirc.
change it for the aforementioned or remove this addition.